scripts/deploy_gke.sh

```diff
@@ -93,6 +93,13 @@ RELEASE=${ENV//_/-}
 IMAGE_ERR=$(gcloud container images describe gcr.io/datcom-ci/datacommons-mixer:"$TAG" > /dev/null ; echo $?)
 if [[ "$IMAGE_ERR" == "1" ]];  then ./scripts/push_binary.sh "$TAG"; fi
 
+PENDING=$(helm history $RELEASE | grep "pending")
+if [[ -n "$PENDING" ]]; then
+  echo "Found pending helm release, deleting"
+  echo "$PENDING"
```

Comment:
nit: echo "Found pending help release $RELEASE, deleting ($PENDING)"

?

Reply:
that's better, thanks!